process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
```

Comment:
s/TOC/CNCF/ ?
Is there a difference here? Can the TOC no longer recommend it but the CNCF continue to be ok with it?
I'm thinking this should CNCF not TOC since the TOC is just acting on behalf of the CNCF.

Reply:
probably better to qualify `project` -> `project under the governance of the TOC`? 

since there are repositories at CNCF that do not come under the TOC (example - https://github.com/cncf/cnf-testbed)